src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -209,7 +209,7 @@ public static int getMenuTextHeightPx(ArrayList<String> menuOptions, int scaledC
   {
     // Height of the letters plus 1 (for buffer between menu options), times the number of entries,
     // minus 1 because there is no buffer after the last entry.
-    return (scaledCharHeightPx + SpriteOptions.getDrawScale()) * menuOptions.size() - SpriteOptions.getDrawScale();
+    return (scaledCharHeightPx + 1) * menuOptions.size() - 1;
```

Comment:
This doesn't give quite enough height, I think.
![image](https://user-images.githubusercontent.com/8674241/59555766-148d4f80-8f7d-11e9-9b1e-5e7f340314e4.png)

Reply:
Yeah, I noticed that. Unfortunately, fixing that bleeds changes into... everything. I was semi-planning to make a separate PR to remove all the drawScale factors except when drawing to the window.

Reply:
What if I just extend the transparent sprite for the text by one line for now, and fix it when we properly address draw-scaling?

Reply:
If we're gonna be fixing the draw scale stuff soonishly (as in, before the next release), I think it's reasonable to just leave it as is.

Reply:
Depends on when the next release is maybe, but yeah, soon (tm) ishly.